packages/perseus/src/widgets/radio/__stories__/choice-icon.stories.jsx

```diff
@@ -1,161 +1,242 @@
-// currently defined, so we give them values anyway.
-const satObject = {
     crossedOut: false,
     focused: false,
     hovered: false,
     pressed: false,
-    checked: false,
-    correct: true,
+    correct: false,
+    showCorrectness: false,
+    product: "library",
     multipleSelect: false,
-    previouslyAnswered: false,
-    product: "sat",
     reviewMode: false,
-    showCorrectness: false,
+    previouslyAnswered: false,
+    transparentBackground: false,
 };
 
-export const Position0Sat = (args: StoryArgs): React.Node => {
-    return <ChoiceIcon {...satObject} pos={0} />;
+const defaultSATProps = {
+    ...defaultProps,
+    product: "sat",
 };
 
-export const Position1CrossedOutSat = (args: StoryArgs): React.Node => {
-    return <ChoiceIcon {...satObject} pos={1} crossedOut={true} />;
+export default {
+    title: "Perseus/Widgets/Radio/Choice Icon",
+    argTypes: {
+        pos: {
+            control: {type: "number", min: 0, max: 25, step: 1},
+        },
+        product: {
+            options: ["library", "sat"],
+            control: {type: "radio"},
+        },
+    },
+    args: defaultProps,
 };
 
-export const Position2SelectedSat = (args: StoryArgs): React.Node => {
-    return (
-        <ChoiceIcon {...satObject} pos={2} checked={true} crossedOut={false} />
-    );
+export const Interactive = (args: StoryArgs): React.Node => {
+    return <ChoiceIcon {...args} />;
 };
 
-export const Position2SelectedReviewModeSat = (args: StoryArgs): React.Node => {
+export const Default = (args: StoryArgs): React.Node => {
     return (
-        <ChoiceIcon
-            {...satObject}
-            pos={2}
-            checked={true}
-            crossedOut={false}
-            reviewMode={true}
-        />
+        <>
+            <p>Library</p>
+            <ChoiceIcon {...defaultProps} />
+            <ChoiceIcon {...defaultProps} multipleSelect={true} />
+            <p>SAT</p>
+            <ChoiceIcon {...defaultSATProps} />
+            <ChoiceIcon {...defaultSATProps} multipleSelect={true} />
+        </>
     );
 };
 
-export const Position2SelectedReviewModeCrossedOutSat = (
-    args: StoryArgs,
-): React.Node => {
+export const Focused = (args: StoryArgs): React.Node => {
     return (
-        <ChoiceIcon
-            {...satObject}
-            pos={2}
-            checked={false}
-            crossedOut={true}
-            reviewMode={true}
-        />
+        <>
+            <p>Library</p>
+            <ChoiceIcon {...defaultProps} focused={true} />
+            <ChoiceIcon
+                {...defaultProps}
+                focused={true}
+                multipleSelect={true}
+            />
+            <p>SAT</p>
+            <ChoiceIcon {...defaultSATProps} focused={true} />
+            <ChoiceIcon
+                {...defaultSATProps}
+                focused={true}
+                multipleSelect={true}
+            />
+        </>
     );
```

Comment:
**suggestion:** since you're repeating this a number of times, it might be worth creating a template component that you could reuse.

Reply:
I did try that originally, but then I noticed that in Storybook it obfuscates the code examples so you don't get this handy feature:

<img width="1011" alt="Screen Shot 2022-10-04 at 11 30 42 AM" src="https://user-images.githubusercontent.com/16308368/193874598-3fdaea40-52a3-4e7e-bddb-f063e86459dc.png">

Instead it just showed the wrapper/template component. I figured it'd be better to have more useful stories than cleaner story code.

Reply:
That makes sense.  I wish there was a way to make that happen without having to copy/paste the thing.